lib/anonymizer.js

```diff
@@ -0,0 +1,324 @@
+const multiparty = require('multiparty');
+const cheerio = require('cheerio');
+
+// load anonymizerConfig
+//const configPath = path.join(__dirname, 'anonymizerConfig.json');
+//const anonymizationConfig = JSON.parse(fs.readFileSync(configPath, 'utf8'));
+
+const emailRegex = /([a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,6})/g;
+const fullNameRegex = /\b([A-Z][a-z]+ [A-Z][a-z]+)\b/g; 
+const singleNameRegex = /\b([A-Z][a-z]+)\b/g; 
+const addressRegex = /\b(\d{1,5}\s\w+(?:\s\w+)*(?:\s(Street|St|Avenue|Ave|Boulevard|Blvd|Road|Rd|Lane|Ln|Drive|Dr|Court|Ct|Circle|Cir))?)\b/g;
+const phoneRegex = /(\+?\d{1,4}[\s-]?)?(\(?\d{3}\)?[\s-]?)?[\d\s-]{7,10}/g;
+const urlRegex = /https?:\/\/[^\s/$.?#].[^\s]*/g;
+const stateRegex = /\b[A-Z]{2}\b/g;
+const zipRegex = /\b\d{5}(?:-\d{4})?\b/g;
+const passwordRegex = /(?<=name="password"[\s\S]*?\r\n\r\n)([A-Za-z0-9]+[A-Za-z0-9]*|[A-Za-z0-9]*[A-Za-z0-9]+)/g;
+
+//default
+let anonymizationEnabled = true;
```

Comment:
@AJun01 anonymization should be not enabled by default. For example if I run the simple-request.js test from examples, despite having NOT set `trafficSimulator.setAnonymization(true);` , the feature is on.